### PR TITLE
More/better comments - fixes #92

### LINKED ISSUE #92: Harmonize class names for request & response DTOs
If there was a 'low priority' label I would have tagged this issue with it.

### PR DESCRIPTION
**Pull request by frankhuster**
Busy work...
Changing the title didn't seem to do it, so I'm adding the magic words below.
Fixes #92

### CODE REVIEW

File: api/src/main/java/org/motechproject/nms/api/web/KilkariController.java [hunk 1]
```diff
@@ -78,7 +76,7 @@ public InboxResponse getInboxDetails(@RequestParam(required = false) Long callin
     }
 
     private void validateSaveInboxCallDetailsContent(StringBuilder failureReasons,
-                                                     Set<InboxCallDetailsRequestCallData> content) {
+                                                     Set<CallDataRequest> content) {
```
Comment (koshalt) [inline api/src/main/java/org/motechproject/nms/api/web/KilkariController.java]: There isn't a CallData request per se, right? CallData object is just part of a list inside of CallDetailsRequest?

File: api/src/main/java/org/motechproject/nms/api/web/UserController.java [hunk 2]
```diff
@@ -143,7 +142,7 @@ private ResponseUser getFrontLineWorkerResponseUser(String serviceName, Long cal
             service = Service.MOBILE_KUNJI;
         }
 
-        user = new FrontLineWorkerUser();
+        user = new FlwUserResponse();
```
Comment (koshalt) [inline api/src/main/java/org/motechproject/nms/api/web/UserController.java]: Just curious, why is this being split from the declaration in line 133?

**Comment by koshalt** [comment]
Builds and tests fine. The comments shouldn't be blocking and feel free to merge once you get back

File: api/src/main/java/org/motechproject/nms/api/web/KilkariController.java (hunk 1 above)
Comment (frankhuster) [inline api/src/main/java/org/motechproject/nms/api/web/KilkariController.java]: Yep

File: api/src/main/java/org/motechproject/nms/api/web/UserController.java (hunk 2 above)
Comment (frankhuster) [inline api/src/main/java/org/motechproject/nms/api/web/UserController.java]: Don't know. But good point, I fixed it.

**Comment by frankhuster** [comment]
@koshalt blessed this change, merging it.

**Pull request merged**